mms/tests/README.md

```diff
@@ -26,14 +26,13 @@ python -m pytest mms/tests/unit_tests/
 To get the coverage report of unit tests, you can run :
 
 ```bash
-python -m pytest --cov-report  term-missing --cov=mms/ --ignore mms/tests/integration_tests/
+python -m pytest --cov-report term-missing --cov=mms/ mms/tests/unit_tests/
```

Comment:
This line is needed.... otherwise integration_tests will be included in the coverage report.

Reply:
I'm removing integration_tests folder, no need to ignore a non-existing folder any more.